src/widget/HomePage/HomePage.tsx

```
@@ -31,15 +32,22 @@ export default function HomePage() {
     }
     return false;
   });
+  
+  const [isClient, setIsClient] = useState(false);
 
   const nextSectionRef = useRef<HTMLDivElement>(null);
   const justUnlockedRef = useRef(false);
   const scrollTimerRef = useRef<ReturnType<typeof setTimeout>>(undefined);
   const relockTimerRef = useRef<ReturnType<typeof setTimeout>>(undefined);
   const isLockedRef = useRef(isLocked);
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)